src/overlays/actors/ovl_En_Rz/z_en_rz.h

```diff
@@ -2,14 +2,36 @@
 #define Z_EN_RZ_H
 
 #include "global.h"
+#include "assets/objects/object_rz/object_rz.h"
+
+#define EN_RZ_GET_PARAM_8000(thisx) ((thisx)->params & 0x8000)
+#define EN_RZ_GET_PARAM_F(thisx) ((thisx)->params & 0xF)
+#define EN_RZ_GET_PATH(thisx) (((thisx)->params & 0x7E00) >> 9)
 
 struct EnRz;
 
 typedef void (*EnRzActionFunc)(struct EnRz*, PlayState*);
 
 typedef struct EnRz {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x2EC];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[OBJECT_RZ_LIMB_MAX];
+    /* 0x188 */ UNK_TYPE1 pad188[0x94];
+    /* 0x2A0 */ Vec3s morphTable[OBJECT_RZ_LIMB_MAX];
+    /* 0x2A0 */ UNK_TYPE1 pad2A0[0x94];
+    /* 0x3B8 */ Path* path;
+    /* 0x3BC */ s32 unk_3BC;
+    /* 0x3C0 */ ColliderCylinder collider;
+    /* 0x40C */ Actor* unk_40C;
+    /* 0x410 */ s16 unk_410;
+    /* 0x412 */ s16 unk_412;
+    /* 0x414 */ Vec3f shadowPos;
+    /* 0x420 */ u16 unk_420;
+    /* 0x422 */ s16 currentAnimIndex;
+    /* 0x424 */ s16 timer;
+    /* 0x426 */ u16 csAction;
+    /* 0x428 */ u16 unk_428;
+    /* 0x42A */ s16 cutscenes[3];
```

Comment:
```suggestion
    /* 0x42A */ s16 cutscenes[2];
```
Based on the `ARRAY_COUNT(this->cutscenes) - 1` it seems more plausible that this is only 2 entries.

Reply:
good catch